File: Feitos em C++ Parte 2/1963.cpp
#include<iostream>
#include<stdio.h>
using namespace std;
int main(){
    double valorini,valorfini;
    cin>>valorini>>valorfini;
    double porc=((valorfini-valorini)/valorini)*100;
    printf("%.2f%%\n",porc);
    return 0;
}

File: Feitos em Java/1963.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);
        double valorini=entrada.nextDouble();
        double valorfini=entrada.nextDouble();
        double porc=((valorfini-valorini)/valorini)*100;
        System.out.printf("%.2f%%\n",porc);
    }
}